client/app/styles/main.styl

```diff
@@ -392,11 +388,11 @@ body
             background-color orange + 10%
 
 
-@media screen and (-webkit-min-device-pixel-ratio:0)
+@media screen and (-webkit-min-device-pixel-ratio: 0)
```

Comment:
Why do we use a `-webkit` property here? Do we target only specific browsers for a hack with this set of rules?